src/types.ts

```diff
@@ -920,6 +920,17 @@ export interface ListenOptions {
    */
   includePreviousRevision?: boolean
 
+  /**
+   * Whether to include events for drafts and versions. As of API Version >= v[TBD], only events
+   * for published documents will be included by default.
+   * If you need events from drafts and versions, set this to `true`.
+   * Note: Keep in mind that additional document variants may be introduced in the future, so it's
```

Comment:
this assumes parity with `perspective=raw`, but given that the param is explicitly named includeAll*Versions*, I guess in theory we could introduce new variants in the future without including them behind this flag.

Reply:
Right, that is possible. We'll have to consider any further developments in this area very carefully.